packages/hs-react-ui/src/components/Button/Button.tsx

```diff
@@ -147,18 +162,20 @@ const Button = ({
   elevation = 0,
   variant = ButtonVariants.fill,
   type = ButtonTypes.button,
-  color = colors.grayDark,
+  color,
   onClick,
   LoadingBar = StyledProgress,
 }: ButtonProps): JSX.Element | null => {
   const hasContent = Boolean(children);
+  const grayDark = useColors('grayDark');
+  const containerColor = color || grayDark;
```

Comment:
Choosing dark grey for their default makes sense but is definitely a stylistic choice... but the only other option would be to make color required which isn't "fun for the whole family right out of the box" so I guess this is best :-/

Reply:
I didn't make any opinionated changes in this PR. This is making sure to handle the old default case on color from the props. 

Totally agree with you, the dark button seems weird. These types of concerns seem like something where we should put up all of our components on one page with the most minimal configuration possible and see what sticks out and address them there.

Reply:
agreeb